### PR TITLE
Edit typos in docs

### PR DESCRIPTION
**Pull request by sangyeol-kim**
Fixed the example code of displayName.

### CODE REVIEW

File: docs/docs/Documenting.md [hunk 1]
```diff
@@ -408,8 +408,8 @@ For instance, if the display name is set as
 
 ```js
 /**
- * @displayName Wonderful Button
- **/
+* @displayName Wonderful Button
```
Comment (elevatebart) [inline docs/docs/Documenting.md]: Why removing the spaces ?

**Comment by elevatebart** [comment]
Can you please update your changelog in git and follow the angular pattern?

`git rebase -i master` can be useful or `git commit --amend`

File: docs/docs/Documenting.md (hunk 1 above)
Comment (sangyeol-kim) [reply docs/docs/Documenting.md]: @elevatebart Ah, is this the pattern used in the angular? I was learning Vue-Styleguidist and thought this was a typo.

**Comment by elevatebart** [comment]
Thank you Kim, 
I believe this was a mistake.
if you feel like I missed something do not hesitate to reopen this PR

**Pull request closed without merging**